content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
 
-  ```sh
-  [root@ ~]# kubectl get pods -A | grep kmesh
-  kmesh-system   kmesh-l5z2j                                 1/1     Running   0          117m
-  ```
+```console
+kubectl describe po httpbin-65975d4c6f-96kgw
+Name:             httpbin-65975d4c6f-96kgw
+Namespace:        default
+Priority:         0
+Service Account:  httpbin
+Node:             kmesh-control-plane/172.18.0.2
+Start Time:       Fri, 26 Apr 2024 11:54:03 +0800
+Labels:           app=httpbin
+                  pod-template-hash=65975d4c6f
+                  version=v1
+Annotations:      kmesh.net/redirection: enabled
+Status:           Running
+IP:               10.244.0.21
+IPs:
+  IP:           10.244.0.21
+Controlled By:  ReplicaSet/httpbin-65975d4c6f
+```
 
-- View the running status of kmesh service
+## Test Sample Applications
 
-  ```sh
-  [root@master mod]# kubectl logs -f -n kmesh-system kmesh-l5z2j
-  time="2024-02-19T10:16:52Z" level=info msg="service node sidecar~192.168.11.53~kmesh-system.kmesh-system~kmesh-system.svc.cluster.local connect to discovery address istiod.istio-system.svc:15012" subsys=controller/envoy
-  time="2024-02-19T10:16:52Z" level=info msg="options InitDaemonConfig successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="bpf Start successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="controller Start successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="command StartServer successful" subsys=manager
-  time="2024-02-19T10:16:53Z" level=info msg="start write CNI config\n" subsys="cni installer"
-  time="2024-02-19T10:16:53Z" level=info msg="kmesh cni use chained\n" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="Copied /usr/bin/kmesh-cni to /opt/cni/bin." subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="kubeconfig either does not exist or is out of date, writing a new one" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="wrote kubeconfig file /etc/cni/net.d/kmesh-cni-kubeconfig" subsys="cni installer"
-  time="2024-02-19T10:16:54Z" level=info msg="command Start cni successful" subsys=manager
-  ```
+After the applications have been manage by kmesh, we need to test that they are still working properly.
 
+```console
+k exec deploy/sleep -- curl -s 10.244.0.21 | grep -o "<title>.*</title>"
+<title>httpbin.org</title>
+```
 
+Note: 10.244.0.21 is the IP of httpbin
 
-## Compile and Build
+## Clean Up
 
-### Source code compilation
+If you don't want to use kmesh to govern the application anymore, you can delete the labels on the namespace and restart the pod.
 
-- Code download
+```console
+kubectl label namespace default istio.io/dataplane-mode-
+kubectl delete po httpbin-65975d4c6f-96kgw sleep-7656cf8794-8tp9n
+kubectl describe po httpbin-65975d4c6f-h2r99
 
-  ```sh
-  [root@ ~]# git clone https://github.com/kmesh-net/kmesh.git
-  ```
+Name:             httpbin-65975d4c6f-h2r99
```

Comment:
Mayb we could grep kmesh related annotaions to prove it was not taken over by Kmesh.

It's better to keep content simple